CRM/Civicase/Hook/BuildForm/CaseCategoryFormLabelTranslationForChangeCase.php

```diff
@@ -57,7 +57,7 @@ private function translateFormLabels(CRM_Core_Form $form) {
    *   For Elements array.
    */
   private function translateLabel($element) {
-    $label = ts($element->getLabel());
+    $label = $element->getLabel();
```

Comment:
This function is used to change relevant form labels for the `Change Case Type` , `Change Case Status` and `Link Cases` to the respective case category equivalent.
I know ts function should only be used with string values but in this case, existing functionality will be broken as this serves a purpose.

Reply:
Fixed, created a new function as discussed.